src/models/Message.model.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+import { DataTypes } from 'sequelize';
+import { sequelize } from '../db.js';
+
+export const Message = sequelize.define(
+  'message',
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    roomId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    text: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    time: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
```

Comment:
It's a better practice to store time as a TIMESTAMP or DATETIME data type, rather than a string. This allows for better manipulation and comparison of date and time values. Consider changing the data type of this field to `DataTypes.DATE` or `DataTypes.DATEONLY` if you don't need to store time.